metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -120,23 +140,19 @@ object Embedded {
       s"mtags_${ScalaVersions.dropVendorSuffix(info.getScalaVersion)}",
       BuildInfo.metalsVersion
     )
-    val needsFullClasspath = !scalac.isSemanticdbEnabled
+    val semanticdbJars = scalac.getOptions.asScala.collect {
+      case opt
+          if opt.startsWith("-Xplugin:") &&
+            opt.contains("semanticdb-scalac") &&
+            opt.contains(BuildInfo.scalametaVersion) =>
```

Comment:
```suggestion
            opt.contains(BuildInfo.semanticdbVersion) =>
```

although they are the same right now, I feel it's more appropriate to use the more specific one

Reply:
Good catch! Thanks!